README.md

```diff
@@ -1,36 +1,265 @@
+
+| Description                                 | Image                | Notes |
+| ------------------------------------------- | -------------------- | :---: |
+| 1x Pro Micro & 18x header pins             | ![Photo of Pro Micro with header pins](https://i0.wp.com/mechwild.com/wp-content/uploads/2020/12/pro_micro.png?resize=200%2C145) | :warning: **A Pro Micro[^1] is *required*, unlike the other items on this list!** :warning: |
+| 1x RGB LED strip (8 LEDs)                   | (unavailable)        | SMD LEDs will not work for this build.      |
+| 90x 0305 Mill-Max sockets                   | (unavailable)        | Used for making the switches hot-swappable. |
+| 90x 7305 Mill-Max sockets                   | (unavailable)        | Used for making the switches hot-swappable. More expensive and more difficult to solder. |
+| SIP socket (18x positions) & 18x brass pins | ![Photo of SIP socket with Mill-Max pins](https://i0.wp.com/mechwild.com/wp-content/uploads/2021/10/sockets_pins.png?resize=200%2C200) | Used to socket the Pro Micro, which makes it removable without desoldering. |
+| 1x Diode Bender                             | (unavailable)        | Helps you [bend multiple diodes at a time consistently](https://mechwild.com/guides/general/diodes/). |
+
+### Step 1: Program, Test, and Solder Pins on Pro Micro[^1]
+
+:bulb: If you run into trouble with any of this, don't hesitate to stop by [MechWild's Discord](https://discord.gg/nfxHnsm) and ask for assistance in #keyboard-help. It is much better to triple-check than to have to desolder a Pro Micro!
+
+#### :warning: Planning to use an Elite-C? :warning:
+
+**The OLED will not work out of the box on Elite-Cs.** Its `VCC` (power) pin is wired to [`RAW` on a typical Pro Micro](https://pbs.twimg.com/media/CnZcb2sWAAADDSa?format=jpg&name=large). The Elite-C [puts `B0` in this position instead](https://deskthority.net/wiki/images/7/7d/Elite-C-black.png).
+
+**Therefore:** if using an Elite-C, do not solder the `B0` pin. Solder a wire connecting `VBUS` or `VCC` on the Elite-C to `VCC` on the OLED instead. If using pin headers, breaking off an extra pin so `B0` can't accidentally be soldered would be a good idea.[^2]
+
+<!-- TODO: most of the info below SHOULD be migrated to a general guide that also contains detailed info on known problems and specific troubleshooting steps. (e.g. power-only cables, USB-C Alt Mode problems, "NO DRIVER" for flashing on Windows...) -->
+
+#### Program Controller
+
+:warning: **You must program the controller with BDE Rev2 firmware before it will work as a keyboard.** :warning:  
+Don't skip this step; you're only postponing the inevitable, and if it turns out your controller is defective you'll regret it.
+
+When you plug the controller in before flashing, it should show up to your OS as a new device:
+
+| Windows                                  | Mac                              | Linux                           |
```

Comment:
Since these are all names of operating systems, you might consider changing "Mac" to "macOS".